src/components/VideoCard/VideoCard.jsx

```diff
@@ -17,11 +21,15 @@ function VideoCard({ video, deleteFromHistory = false }) {
     }
   };
 
-  const removeFromHistoryHandler = (e) => {
+  const deleteHandler = (e) => {
     e.preventDefault();
     e.stopPropagation();
```

Comment:
suggestion: if you are writing `event` then use that convention or only `e` then use that, anyone way not both

Reply:
yes. my bad. I will keep it consistent